doc/man7/EVP_PKEY-ML-KEM.pod

```diff
@@ -138,7 +183,7 @@ recognised on input.
 
 =back
 
-=item C<ml-kem.output_formats> (B<OSSL_PKEY_PARAM_ML_KEM_INPUT_FORMATS>) <UTF8 string>
+=item C<ml-kem.output_formats> (B<OSSL_PKEY_PARAM_ML_KEM_INPUT_FORMATS>)
```

Comment:
_OUTPUT_FORMATS>

Reply:
On line 200 why do you only specify the order of the first 2 default values...

````default C<seed-priv> is listed first and C<priv-only> second.````

Reply:
Pushed an update that expands the description.